rouille_proc_macro/src/lib.rs

```diff
@@ -83,6 +83,58 @@ rouille_compilogenese::rouille! {
             "Ponctuation" => "Punct",
             "Litéral" => "Literal",
             "macro_procédurale" => "proc_macro",
+            "LectureBufferisée" => "BufRead",
+            "Lecture" => "Read",
+            "Écriture" => "Write",
+            "entrée_sortie" => "io",
+            "entrée" => "stdin",
+            "sortie" => "stdout",
+            "lire_ligne" => "read_line",
+            "écrire_ligne" => "write_line",
+            "écrire" => "write",
+            "est_vide" => "is_empty",
+            "est" => "is",
+            "verrouiller" => "lock",
+            "octets" => "bytes",
+            "chaîne" => "str",
+            "lire_exactement" => "read_exact",
+            "lire_jusqu'à" => "read_until",
+            "sépare" => "split",
+            "en_tant_que" => "as",
+            "Entier" => "Integer",
+            "ensuite" => "then",
+            "ensuite_valeur" => "then_some",
+            "déballer_ou" => "unwrap_or",
+            "déballer_ou_avec" => "unwrap_or_else",
+            "en" => "into",
+            "depuis_chaîne" => "from_str",
+            "Depuis" => "From",
+            "bien" => "ok",
+            "bien_ou" => "ok_or",
+            "bien_ou_avec" => "ok_or_else",
+            "en_tant_que_déréférencé" => "as_deref",
+            "en_tant_que_déréférencé_mutable" => "as_mut_deref",
+            "itérable" => "iter",
+            "itérable_mut" => "iter_mut",
+            "énumérer" => "enumerate",
+            "filtre" => "filter",
+            "filtre_applique" => "filter_map",
+            "applique" => "map",
+            "applique_ou" => "map_or",
+            "applique_ou_avec" => "map_or_else",
+            "pour_chaque" => "for_each",
+            "essayer_pour_chaque" => "try_for_each",
+            "applatir" => "flatten",
+            "compter" => "count",
+            "trouve" => "find",
+            "trouve_en_appliquant" => "find_map",
+            "tous" => "all",
```

Comment:
Nous sommes en 2024, pensez printemps les ami·es.

```suggestion
            "tous·tes" => "all",
```

Reply:
je propose plutot d'utiliser, tous ou toutes separement pusique cette notation est complexe a utiliser sur un ordinateur